app/src/main/java/com/example/trempel/ui/pdp/PdpViewModel.kt

```diff
@@ -34,11 +39,26 @@ internal class PdpViewModel @Inject constructor(
             }
             .subscribe({ response ->
                 _product.value = response
+                addRecentlyViewedProduct(response)
             }, { error ->
                 _errorLiveData.value = error.message
             })
     }
 
+    fun getRecentlyViewedProduct(idProduct: Int) {
+        disposable = roomRepository.getLatestRecentlyViewed(idProduct)
```

Comment:
you're reassigning disposable each time, please change to `CompositeDisposable` and call `+=` instead of `=`

Reply:
done